### ISSUE TITLE
Add instruction to run ember build and ember test

### ISSUE THREAD

**Issue by jrhorn424** [ga-wdi-boston/ember-repo-template#10]
Before running ember server.

**Comment by raq929** [comment]
Do you want to add this as an instruction in the README, or automatically run them via grunt when the server is run?

**Comment by gaand** [comment]
I don't think we want to run them automatically.  We don't do that for any other template.

We don't currently use `grunt server` for this.  I'm not sure we should.  We're using `ember-cli` and should probably configure it if possible.

**Comment by raq929** [comment]
OK. I'm confused about what we are configuring it to do.

**Comment by gaand** [comment]
Me too.
